examples/app/ios/README.md

```diff
@@ -26,14 +26,14 @@ This will not be a complete tutorial on how to use `uniffi`, just the bits to ge
 4. Use `uniffi-bindgen` on each file to generate the headers and swift scaffolding.
 
 ```sh
-uniffi-bindgen generate "$INPUT_FILE_PATH" --language swift --out-dir "$DERIVED_FILE_DIR"
+$HOME/.cargo/bin/uniffi-bindgen generate "$INPUT_FILE_PATH" --language swift --out-dir "$DERIVED_FILE_DIR"
```

Comment:
Hrm, it's a shame we can't just pick this up from `$PATH`, but I noticed the same thing. Do you know if there's a way we could tell XCode about the `$PATH` somehow?